CONTRIBUTING.md

```diff
@@ -4,6 +4,74 @@ Watch the introduction video [video \(IT\)](https://www.youtube.com/watch?v=KH8x
 and contact [Andrea Idini](mailto:andrea.idini@gmail.com).
 Priviledge channel for contributions is the telegram group [@scienza](https://t.me/Scienza).
 
+### Styling guide
+This is based on the [Google C++ Styling guide](https://google.github.io/styleguide/cppguide.html), here's the most important things:
+
+- Use `.cpp` (**NOT** `.cc`) as extension for C++ files and `.h` for headers files
+- every `.cpp` file should have an associated `.h` file
+- Use [header guards](https://en.wikipedia.org/wiki/Include_guard) ( `#pragma once` ([rel](https://en.wikipedia.org/wiki/Pragma_once)) can be used, but header guards are preferred)
+- Use Forward Declarations (declare functions without defininf them, basically using the header)
+- Prefer small and focused functions
+- Use inline function for 8 lines long functions or less
+- Do not define implicit conversions. Use the explicit keyword
+- Use a struct only for passive objects that carry data; everything else is a class.
+- Declare variables in a local way whenever is possible (best prefix for a global variable is `//`)
+- Initialize variables when declaring them
+- Align the equal sign when declaring multiple variables
+- readability > functionality, don't be afraid of writing 5 lines of code more
+- Use namespaces, the name should be about the functionality itself
+- **DO NOT** use `using` (e.g. `using namespace std;`)
+- At the end of namespace, comment it's end `} // namespace nsName`
```

Comment:
Unclear

Reply:
"its", but you can leave just "comment".